src/Symfony/Component/Form/Extension/Core/DataTransformer/PercentToLocalizedStringTransformer.php

```diff
@@ -120,7 +120,7 @@ public function reverseTransform($value)
 
         $formatter = $this->getNumberFormatter();
         // replace normal spaces so that the formatter can read them
-        $value = $formatter->parse(str_replace(' ', ' ', $value));
+        $value = $formatter->parse(str_replace(' ', pack('CC', 0xc2, 0xa0), $value));
```

Comment:
I'm fine with both.
BTW, do you know why we do that replacement in the first place? that conversion is not covered by tests.